java/java-impl/src/META-INF/JavaPlugin.xml

```diff
@@ -237,6 +237,9 @@
     <extensionPoint qualifiedName="com.intellij.debugger.asyncStackTraceProvider"
                     interface="com.intellij.debugger.engine.AsyncStackTraceProvider" dynamic="true"/>
 
+    <extensionPoint qualifiedName="com.intellij.debugger.kotlinStackFrameValueContributor"
```

Comment:
Please register extension points in the Kotlin plugin if they're Kotlin-only.
This particular extension point should be registered in `plugins/kotlin/plugin/common/resources/META-INF/jvm-debugger.xml`.

Reply:
Done